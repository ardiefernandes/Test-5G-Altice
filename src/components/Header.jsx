import React from "react";
import MEO_Logo from "./../assets/MEO Logo.svg";

const Header = () => {
  return (
    <header className="app-header">
      <img alt="meo logo" src={MEO_Logo} />
    </header>
  );
};

export default Header;
