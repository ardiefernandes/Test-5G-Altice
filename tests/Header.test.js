import React from "react";
import renderer from "react-test-renderer";
import Header from "./../src/components/Header";

it("correctly renders the header", () => {
  const component = renderer.create(<Header />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
