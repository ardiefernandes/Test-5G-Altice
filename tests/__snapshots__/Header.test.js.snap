// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`correctly renders the header 1`] = `
<header
  className="app-header"
>
  <img
    alt="meo logo"
    src="MEO Logo.svg"
  />
</header>
`;
